src/tsdb/src/tsdbMeta.c

```diff
@@ -734,10 +746,7 @@ void tsdbUpdateTableSchema(STsdbRepo *pRepo, STable *pTable, STSchema *pSchema,
   TSDB_WUNLOCK_TABLE(pCTable);
 
   if (insertAct) {
-    int   tlen = tsdbGetTableEncodeSize(TSDB_UPDATE_META, pCTable);
-    void *buf = tsdbAllocBytes(pRepo, tlen);
-    ASSERT(buf != NULL);
-    tsdbInsertTableAct(pRepo, TSDB_UPDATE_META, buf, pCTable);
+    ASSERT(tsdbInsertNewTableAction(pRepo, pCTable) == 0);
```

Comment:
if NDEBUG is not defined, ASSERT is define as null. Please make sure the behaviour

Reply:
Besides, memory allocation error is possible.

Reply:
fixed